integrations/cohere/src/haystack_integrations/components/embedders/cohere/text_embedder.py

```diff
@@ -138,38 +147,19 @@ def run(self, text: str):
         """
         text = self._prepare_input(text=text)
```

Comment:
text embedder and document embedder are a bit inconsistent here. text embedder has a an extracted method `_prepare_input`, document embedder doesn't. Having it is slightly better in my opinion and I would suggest renaming it to `_validate_input` because the inputs remain unchanged.